server/shared/logger.js

```diff
@@ -4,7 +4,8 @@ const bunyan = require('bunyan');
 
 const logger = bunyan.createLogger({
   name: 'tailor-server',
-  serializers: bunyan.stdSerializers
+  serializers: bunyan.stdSerializers,
+  level: process.env.LOG_LEVEL
```

Comment:
It should default here to `info` if If `LOG_LEVEL` is not set in the `.env` file.

Reply:
@abasic How about dynamic default dependant on `NODE_ENV`?
```js
const logLevel = (() => {
  if(process.env.LOG_LEVEL) return process.env.LOG_LEVEL;
  return process.env.NODE_ENV === 'production' ? 'info' : 'debug';
})();

// ...
level: logLevel
```

Reply:
I have nothing against having a different logging level based on the environment. The reason why I said it should default to `info` was that that is how it works now so I wanted to maintain parity. I assumed there was a reason why it is currently always `info`. We can go this way if nobody has a problem with it.

Reply:
@kronicker Upgraded changing logger level based on `.env`👇 . If you reconsider this approach, let me know here, it will be done :bowtie: .